C05 kks dialogue (#103)
IllusionMods/KKS-Translation: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c05/00/61/translation.txt

//
// Dumped for Koikatsu Sunshine v1.0.1 (KoikatsuSunshine) by KKS_TextDump v1.4.5.0
//

> 「オイル塗り……ふーん、そんなことが出来るんだ」=“Oiling... mm-hmm, you can do that.”

Reviewer: ```suggestion
「オイル塗り……ふーん、そんなことが出来るんだ」=“Oiling... mm-hmm, so you can do that here.”
```

> 「せっかくだし塗っていく？　俺、手伝うよ」=“I'll spread it on you with great care. It'll really help.”

Reviewer: ```suggestion
「せっかくだし塗っていく？　俺、手伝うよ」=“Why don't you try it out? I'll help put them on.”
```
